source/schema.txt

```diff
@@ -338,3 +338,424 @@ To delete a location filter from the map:
 .. |query-build-ref| replace:: :ref:`build-query`
 
 .. |query-build-type| replace:: :ref:`build-query`
+
+Export Your Schema
+------------------
+
+You can export your schema after analyzing it. This is useful for 
+sharing your schema and comparing schemas across collections.
```

Comment:
[note] let's call attention to this in external review to see if there are any other specific use cases we want to highlight.